src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+    public void setColor(String color) {
+        this.color = color;
+    }
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+    public Ball() {
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    private int number;
```

Comment:
It's a good practice to declare class fields at the beginning of the class before any methods or constructors. Please move the declaration of 'number' to the top of the class, right below the 'color' field.